src/Sarif.Viewer.VisualStudio/SCloseSarifLogService.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sarif.Viewer
+{
+    internal interface SCloseSarifLogService
```

Comment:
Why does this interface not start with I and how about a comment what it will be used for?

Reply:
This defines the "service" that is used for various IServiceProvider (ish) techniques used in visual studio. It's a well established pattern.

---
In reply to: [440941253](https://github.com/microsoft/sarif-visualstudio-extension/pull/143#discussion_r440941253) [](ancestors = 440941253)